<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminChat</class>
 <widget class="QMainWindow" name="AdminChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrator Chat </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3" colspan="2">
       <widget class="QListWidget" name="lwBlocked">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pbBlock">
        <property name="text">
         <string>Block</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pbUnblock">
        <property name="text">
         <string>Unblock</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QListWidget" name="lwUnblocked">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leBlock"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="leUnBlock"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbLoadUnblockedUsers">
        <property name="text">
         <string>Load Unblock users</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pbLoadBlockUsers">
        <property name="text">
         <string>Load Block users</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
